01_fib/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+var (
+	num1, num2 = 0, 1
+	sum        = 0
+)
+
+func fib(n int) {
+	fmt.Fprintln(out, num2)
```

Comment:
It's better to decouple fibonacci sequence calculation from IO. https://github.com/anz-bank/go-training/issues/37